src/scripts/main.js

```diff
@@ -18,10 +18,26 @@ const food = {
   },
 };
 
+document.body.id = 'tree';
+
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const list = document.createElement('ul');
+
+  for (const key in data) {
+    const listElement = document.createElement('li');
+
+    listElement.textContent = key;
+
+    if (Object.keys(data[key]).length > 0) {
+      createTree(listElement, data[key]);
```

Comment:
The recursive call to `createTree` should append the newly created list to the `listElement` before making the recursive call. Otherwise, the nested lists might not be correctly appended to their parent list elements.